### ISSUE TITLE
Creating a Taxon by API

### ISSUE THREAD

**Issue by jarekkowol** [Sylius/Sylius#8920]
My request according to http://docs.sylius.org/en/latest/api/taxons.html:
```
curl http://localhost:8000/api/v2/taxons/ \
    -H "Authorization: Bearer NjRmNTk0OTU3YTYxY2Q4YjM0Mjk4YTAwNTRmYTRhMWJmOWU3MGM4MWY5OTJkOWU3NzdiNzJiY2E1OGJjODAwMA" \
    -H "Content-Type: application/json" \
    -X POST \
    --data '
        {
            "code": "toys"
        }
    '
```

Response:
`{"code":400,"message":"Validation Failed","errors":{"children":{"translations":{"children":{"en_US":{"children":{"name":{},"slug":{},"description":{}}},"pl_PL":{"children":{"name":{"errors":["Prosz\u0119 poda\u0107 nazw\u0119 taksonomii."]},"slug":{"errors":["Wprowad\u017a fragment URL kategorii."]},"description":{}}},"pl":{"children":{"name":{},"slug":{},"description":{}}}}},"images":{},"code":{},"parent":{}}}}`

Any idea?

**Comment by igormukhingmailcom** [comment]
Looks like you should specify name and slug.

**Comment by jarekkowol** [comment]
If I add name or slug, response has error like unexpected params

3 lis 2017 17:58 "Igor Mukhin" <notifications@github.com> napisał(a):

**Comment by GSadee** [comment]
Even if you do sth like that?:
```
$ curl http://demo.sylius.org/api/v1/taxons/ \
    -H "Authorization: Bearer SampleToken" \
    -H "Content-Type: application/json" \
    -X POST \
    --data '
        {
            "code":"toys",
            "translations":{
                "en_US": {
                    "name": "Toys",
                    "slug": "category/toys"
                }
            }
            ]
        }
    '
```

**Comment by lchrusciel** [comment]
Name and slug in the default locale is always required. We should update the docs probably. And @GSadee proposal should work.

**Comment by pamil** [comment]
Fixed by #9755.
